File: Python/proyecto.py
class Account:
    def __init__(self, id, nombre, document, email,password):
        self.id = id
        self.nombre = nombre
        self.document = document
        self.email = email
        self.password = password


class Car:
    def __init__(self, id, license, driver,passenger ):
        self.id = id
        self.license = license
        self.driver = driver
        self.passenger = passenger


class Payment:
    def __init__(self, id,):
        self.id = id


class Route:
    def __init__(self, id,init, end):
        self.id = id
        self.init = init
        self.end = end



if __name__ == "__main__":
    pass

File: Javascript/proyecto.js
class Account {
    constructor( id, nombre, document, email,password) {
        this.id = id
        this.nombre = nombre
        this.document = document
        this.email = email
        this.password = password
    }
    
}

class Car {
    constructor(id, license, driver,passenger ){
        this.id = id
        this.license = license
        this.driver = driver
        this.passenger = passenger
    }
}

class Payment {
    constructor( id){
        this.id = id
    }
}

class Route{
    constructor(id,init, start){
        this.id = id
        this.init = init
        this.start = start
    }
}